modules/carto/bundle.ts

```diff
@@ -7,3 +7,7 @@ import * as CartoUtils from '@deck.gl/carto';
 export * from '../layers/bundle/peer-dependency';
 
 export const carto = CartoUtils;
+
+// Export CARTO library for pydeck integration.
+// More info: https://github.com/ajduberstein/pydeck_custom_layer
+globalThis.CartoLibrary = CartoUtils;
```

Comment:
This line is also included in https://github.com/visgl/deck.gl/pull/8799. I've filed the separate PR because the deck.gl v9.0.x patch release could potentially happen before the pydeck changes land.